.github/workflows/test.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       - name: Display go version
         run: go version
       - name: Run all tests
-        run: go test -mod=readonly -timeout 30m -coverprofile=coverage.txt -tags='norace' -covermode=atomic `go list ./... | grep -v simapp`
+        run: go test -mod=readonly -timeout 30m -coverprofile=coverage.txt -tags='norace' -covermode=atomic `go list ./... | grep -E -v 'simapp|e2e'`
```

Comment:
nit: we should instead encapsulate this in a make target.

See the changes I made here: https://github.com/cosmos/gaia/blob/main/Makefile#L167-L205

I don't think it's too much trouble to do that in this PR. WDYT?